<!-- -->
<launch>

	<node name="interface_node" pkg="interface" type="interface_node" output="screen">
	    <param name="utm_topic" value="/ll2utm"/>
		<param name="can2serial_port" value="/dev/ttyUSB0"/>
		<param name="can_baudrate" value="250"/>
	</node>
	
	<node name="auto_drive_node" pkg="auto_drive" type="auto_drive_node" output="screen">
	    <param name="utm_topic" value="/ll2utm"/>  
		<param name="path_file_dir" value="$(find record_path)/data"/>
		<param name="foreSightDis_speedCoefficient" value="1.8"/>
		<param name="foreSightDis_speedCoefficient" value="1.8"/>
		<param name="min_foresight_distance" value="5.0"/>
		<param name="wheel_base" value="2.0"/>
	</node>
	
	<node name="record_path_node" pkg="record_path" type="record_path_node" output="screen">
	    <param name="utm_topic" value="/ll2utm" />
		<param name="file_path" value="$(find record_path)/data/"/>
		<param name="sample_distance" value="0.1"/>
	</node>
	
	<node name="nuogeng_node" pkg="nuogeng" type="nuogeng_node" output="screen">
		<param name="port_name" value="/dev/ttyUSB1"/>
		<param name="is_ll2utm" value="true"/>
		<param name="odom_topic" value="/ll2utm"/>
	</node>
	
</launch>
